x-pack/plugins/apm/public/components/app/ErrorGroupOverview/Watcher/__test__/WatcherButton.test.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import DetailView from '../WatcherButton';
+
+jest.mock('ui/chrome', () => ({
+  addBasePath: path => `myBasePath${path}`
+}));
+
+describe('WatcherButton', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(<DetailView />);
+  });
+
+  it('should render initial state', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should have correct url', () => {
+    const panels = wrapper.find('EuiContextMenu').prop('panels');
+    expect(panels[0].items[1].href).toBe(
+      'myBasePath/app/kibana#/management/elasticsearch/watcher/'
+    );
+  });
```

Comment:
The test here feel a little unnecessary. Let me know if you any suggestions.

Reply:
Yeah I see what you mean but I think it's fine to be a little extra explicit with what we're worried about there with base paths.